pkgs/applications/misc/easystroke/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, fetchFromGitHub, fetchpatch, pkgconfig, boost, dbus-glib, gnome3
+, xorg, help2man, intltool, libsigcxx, wrapGAppsHook}:
+
+stdenv.mkDerivation rec {
+  name = "easystroke-${version}";
+  version = "unstable-2016-07-05";
+
+  src = fetchFromGitHub {
+    owner = "thjaeger";
+    repo = "easystroke";
+    rev = "f7c1614004e9c518bd4f6f4b3a2ddaf23911a5ef";
+    sha256 = "0map8zbnq993gchgw97blf085cbslry2sa3z4ambdcwbl0r9rd6x";
+  };
+
+  nativeBuildInputs = [ help2man pkgconfig wrapGAppsHook ];
+  buildInputs = [
+    boost dbus-glib gnome3.defaultIconTheme gnome3.gtkmm intltool
+    libsigcxx xorg.inputproto xorg.libXtst xorg.xorgserver
+  ];
+
+  patches = [
+    # Upstream development has ceased and some patches are now required to fix the build and avoid crashes:
+    # https://web.archive.org/web/20181217031130/https://github.com/thjaeger/easystroke/wiki/BuildInstructions#Releases
+    #
+    # See also https://github.com/thjaeger/easystroke/pull/10#issuecomment-444132355
+
+    (fetchpatch {
+      # Fix build failure with libsignc++ version 2.5.1 or newer
+      url = "https://github.com/thjaeger/easystroke/pull/9/commits/22b28d25bb696e37e73b4bc641439b3db9f564ed.patch";
```

Comment:
Please change the URL to https://github.com/thjaeger/easystroke/commit/22b28d25bb696e37e73b4bc641439b3db9f564ed.patch in case the PR gets rebased and the ref dropped (not certain if this can happen).